Makefile.toml

```diff
@@ -1305,6 +1305,11 @@ command = "target/${BUILD_MODE_DIR}/risedev-dev"
 args = ["${@}"]
 description = "Clean data and start a full RisingWave dev cluster using risedev-dev"
 
+[tasks.ci-kill-no-logs]
+category = "RiseDev - CI"
+dependencies = ["k", "check-logs", "wait-processes-exit"]
```

Comment:
`check-logs` seems useless if we don't dump them with `logs`?

Reply:
Can still check if the logs are too large.

Reply:
We don't want to upload large log files

Reply:
You're right. `dump` means dumping files into the console. 😄